desktop/install/linux-install.md

```diff
@@ -268,6 +270,47 @@ Docker Desktop for Linux runs a Virtual Machine (VM) for the following reasons:
 
     As such, we have adjusted the default memory available to the VM in DD4L. You can tweak this setting to your specific needs by using the **Memory** slider within the **Settings** > **Resources** tab of Docker Desktop.
 
+## File sharing
+
+Docker Desktop for Linux uses [virtiofs](https://virtio-fs.gitlab.io/) as the
+default (and currently only) mechanism to enable file sharing between the host
+and Docker Desktop VM. In order not to require elevated privileges, without
+unnecessarily restricting operations on the shared files, Docker Desktop runs
```

Comment:
```suggestion
and Docker Desktop VM. In order not to require elevated privileges without
unnecessarily restricting operations on the shared files, Docker Desktop runs
```

Reply:
"without unnecessarily restricting operations on the shared files", is an interjection here, so it should go in between commas?

Reply:
It didn't sound right to me as an interjection, but ok don't bother too much for a comma